bin/start-collectd

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+: ${GRAPHITE_HOST:=}
+: ${GRAPHITE_PORT:="2003"}
+
+set -eu
+
+CONF_TEMPLATE=/etc/collectd/collectd.conf.tpl
+CONF_DEST=/etc/collectd/collectd.conf
+
+HOSTNAME=$(wget -O - --tries 1 --timeout 0.2 http://169.254.169.254/1.0/meta-data/instance-id 2>/dev/null || hostname)
```

Comment:
I don't think this'll work (with `--tries` and `--timeout`) on Alpine, as I think `wget` is the busybox version of `wget` by default. Had a quick noodle and I think the best you'll manage is probably ` wget -O - -T 1 http://...`.